packages/design-system/src/styles/settings/variables/_color.scss

```diff
@@ -109,3 +109,7 @@ $color-background-dialog: $color-white !default;
 $border-color: $color-gray-lighter !default;
 $border-color-dark: $color-gray-dark !default;
 $border-color-inverse: $color-white !default;
+
+// USA Banner
+$usa-banner-background-color: #f0f0f0;
```

Comment:
I think this should be a theme variable, so it should be inside the USA banner scss file.